integration/devmode/devmode_test.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package devmode
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strconv"
+	"syscall"
+	"time"
+
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/hyperledger/fabric/integration/nwo"
+	"github.com/hyperledger/fabric/integration/nwo/commands"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+	"github.com/tedsuo/ifrit"
+	"github.com/tedsuo/ifrit/ginkgomon"
+)
+
+var _ = Describe("Devmode", func() {
+	var (
+		testDir                  string
+		client                   *docker.Client
+		network                  *nwo.Network
+		process                  ifrit.Process
+		chaincode                nwo.Chaincode
+		chaincodeRunner          *ginkgomon.Runner
+		chaincodeProcess         ifrit.Process
+		channelName, chaincodeID string
+		org1peer0                *nwo.Peer
+	)
+
+	BeforeEach(func() {
+		var err error
+		channelName = "testchannel"
+		testDir, err = ioutil.TempDir("", "devmode")
+		Expect(err).NotTo(HaveOccurred())
+
+		client, err = docker.NewClientFromEnv()
+		Expect(err).NotTo(HaveOccurred())
+
+		chaincode = nwo.Chaincode{
+			Name:    "mycc",
+			Version: "0.0",
+			Path:    "github.com/hyperledger/fabric/integration/chaincode/simple/cmd",
+			Ctor:    `{"Args":["init","a","100","b","200"]}`,
+			Policy:  `AND ('Org1MSP.member')`,
+		}
+
+		chaincodeID = chaincode.Name + ":" + chaincode.Version
```

Comment:
Nit: similar to the PR on master, can we define this down where we use it instead of up here?